runtime/isp_run_app.py

```diff
@@ -45,6 +45,9 @@ def main():
     parser.add_argument("-r", "--runtime", type=str, default="bare", help='''
     Currently supported: frtos, sel4, bare (bare metal) (default), stock_frtos, stock_sel4, stock_bare
     ''')
+    parser.add_argument("--arch", type=str, default="rv32", help='''
```

Comment:
We could also autodetect the arch based on `readelf` since we have the binary (this can wait for another PR though)

Reply:
We thought about that, but this parameter is sprinkled through most of the other scripts so for consistency we went this way.  Can make an issue to track this request.

Reply:
https://github.com/draperlaboratory/hope-tools/issues/65